tests/versioned/apollo-server-lambda/apollo-server-lambda-setup.js

```diff
@@ -27,7 +28,10 @@ function setupApolloServerLambdaTests({suiteName, createTests, pluginConfig}, co
     t.beforeEach(async () => {
       const { ApolloServer, gql } = require('apollo-server-lambda')
 
+      agentTesting.temporarySetEnv(t, 'NEW_RELIC_ACCOUNT_ID', 'eeeeee')
```

Comment:
this is using the outer `t` so could be before/outside the beforeEach and is not needed to be called multiple times that way.

if you want tied to the individual test run, a `t` param does get passed to `beforeEach`. 

in callback style, that looks like `t.beforeEach((done, t) => {})` not sure when you are using async...

Reply:
I think async is a copy/pasta fail.  I'll fix it.

Reply:
ah yeah, maybe from when you were doing koa. i shoulda checked for that.

so yeah, use callback style and invoke the `done()` and use the `t` and that will cleanup after each individual test. which isn't necessary, it could be before all, but since the rest of the setup/teardown is per test might as well stay consistent.